This patch will remove some static definitions which clash with extern.

diff -urN BitchX/source/timer.c bitchx/source/timer.c
--- BitchX/source/timer.c       2003-04-11 03:09:07.000000000 +0200
+++ bitchx/source/timer.c       2006-07-19 14:01:42.000000000 +0200
@@ -148,7 +148,7 @@
  * This is put here on purpose -- we dont want any of the above functions
  * to have any knowledge of this struct.
  */
-static TimerList *PendingTimers;
+TimerList *PendingTimers;
 static char *schedule_timer (TimerList *ntimer);
 
 static char *current_exec_timer = empty_string;

--- BitchX/source/ctcp.c.orig	2003-05-27 09:00:22.000000000 +0200
+++ BitchX/source/ctcp.c	2006-07-19 14:34:09.000000000 +0200
@@ -176,7 +176,7 @@
 
 /* CDE do ops and unban logging */
 
-static char	*ctcp_type[] =
+char	*ctcp_type[] =
 {
 	"PRIVMSG",
 	"NOTICE"
--- BitchX/source/term.c.orig	2003-04-11 03:09:07.000000000 +0200
+++ BitchX/source/term.c	2006-07-19 14:34:44.000000000 +0200
@@ -92,7 +92,7 @@
 #endif
 
 extern  char    *getenv();
-extern	char	*tparm();
+/* extern	char	*tparm(); */
 
 /*
  * The old code assumed termcap. termcap is almost always present, but on
